tests/app/service/test_discovery.py

```diff
@@ -199,6 +200,7 @@ def test_create_ai_apps_model(discovery, mock_pebblo_server_version):
         "lastUsed": datetime.datetime(2024, 1, 1, 0, 0, 5),
         "pebbloServerVersion": "x.x.x",
         "pebbloClientVersion": "0.1.1",
+        "clientVersion": {"name": "langchain_community_version", "version": "x.x.x"},
```

Comment:
nit: `name` should be just pkg name i.e. `langchain_community`
```suggestion
        "clientVersion": {"name": "langchain_community", "version": "x.x.x"},
```